x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/expression/predicate/operator/comparison/BinaryComparison.java

```diff
@@ -49,4 +49,6 @@ protected Pipe makePipe() {
     public static Integer compare(Object left, Object right) {
         return Comparisons.compare(left, right);
     }
+
+    public abstract BinaryComparison reverse();
```

Comment:
Add javadoc regarding the purpose of the method.

Reply:
javadoc added.